cli/cmd/install/install.go

```diff
@@ -85,21 +81,26 @@ func helmInstall(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("could not install The Observability Stack: %w", err)
 	}
 
-	// TODO(paulfantom): Remove deprecated flags post 0.10.0 release
+	// TODO(paulfantom): Remove flags notices post 0.12.0 release
 	if cmd.Flags().Changed("tracing") {
-		fmt.Println("DEPRECATED flag used: 'tracing'. This flag will be removed in future versions of tobs. Feature is now enabled by default and if you want to disable deployment of opentelemetry-operator, change value of opentelemetryOperator.enabled")
+		fmt.Println("REMOVED flag used: 'tracing'. Feature is now enabled by default and if you want to disable deployment of opentelemetry-operator, change value of opentelemetryOperator.enabled")
+		os.Exit(1)
 	}
 	if cmd.Flags().Changed("skip-wait") {
-		fmt.Println("DEPRECATED flag used: 'skip-wait'. This flag will be removed in future versions of tobs. Feature is now disabled by default to allow smooth installation of opentelemetry components")
+		fmt.Println("REMOVED flag used: 'skip-wait'. This flag will be removed in future versions of tobs. Feature is now disabled by default to allow smooth installation of opentelemetry components")
```

Comment:
```suggestion
		fmt.Println("REMOVED flag used: 'skip-wait'. Feature is now disabled by default to allow smooth installation of opentelemetry components")
```

small nit: The flag is already removed, so let's remove this line as well (as done in other notices).